.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libsodium-sys/libsodium"]
+	path = libsodium-sys/libsodium
+	url = https://github.com/jedisct1/libsodium.git
```

Comment:
Is it v1.0.18 or master?

Reply:
[The submodule](https://github.com/sodiumoxide/sodiumoxide/pull/393/files#diff-56e707f409b4a26a091f2f02232abafe) is at [940ef4](https://github.com/jedisct1/libsodium/tree/940ef42797baa0278df6b7fd9e67c7590f87744b) which corresponds to [1.0.18-RELEASE](https://github.com/jedisct1/libsodium/releases/tag/1.0.18-RELEASE), so this should be v1.0.18. (There is another tag called [1.0.18](https://github.com/jedisct1/libsodium/releases/tag/1.0.18) from the say date, but it did not advertise a tarball.)

Reply:
Thanks